.github/workflows/ci.yml

```diff
@@ -18,6 +18,8 @@ jobs:
       run: npm run lint
     - name: Test
       run: npm run test
+    - name: Buils
```

Comment:
typo: `Buils` > `Build`?

Reply:
BTW why we need this?

Reply:
Done, for details of the change see https://edx-internal.slack.com/archives/CD3L7U2SZ/p1636463279080800?thread_ts=1636461695.073300&cid=CD3L7U2SZ